mindocr/models/utils/misc.py

```diff
@@ -0,0 +1,12 @@
+from packaging import version
+
+import mindspore as ms
+from mindspore.ops.primitive import constexpr
+
+
+@constexpr
+def ms_version_is_large_than_2_0():
```

Comment:
`def is_ms_version_2():`?
And maybe move it somewhere outside `models/utils`? I think this function is useful and can be applied beyond models scope.

Reply:
updated